.github/workflows/build_and_test.yml

```diff
@@ -11,10 +11,10 @@ jobs:
       run: |
         sudo apt update -qq
         sudo apt install -y default-jdk
-    - uses: ros-tooling/setup-ros@3b3f060
+    - uses: ros-tooling/setup-ros@b1c8132
```

Comment:
Looks like we can use the release tag [0.0.24](https://github.com/ros-tooling/setup-ros/tree/0.0.24).

Reply:
Fixed in 406844e.